Name: CVE-2013-2099
Status: Candidate
URL: http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-2099
Phase: Assigned (20130219)
Category: 
Reference: MLIST:[oss-security] 20130515 Re: CVE Request (minor) -- Python 3.2: DoS when matching  certificate with many '*' wildcard characters {was: CVE Request  (minor) --  python-backports-ssl_match_hostname: Denial of service when matching  certificate with many '*' wildcard characters }
Reference: URL:http://www.openwall.com/lists/oss-security/2013/05/16/6
Reference: MISC:https://bugzilla.redhat.com/show_bug.cgi?id=963260
Reference: CONFIRM:http://bugs.python.org/issue17980
Reference: REDHAT:RHSA-2014:1690
Reference: URL:http://rhn.redhat.com/errata/RHSA-2014-1690.html
Reference: REDHAT:RHSA-2016:1166
Reference: URL:https://access.redhat.com/errata/RHSA-2016:1166
Reference: UBUNTU:USN-1983-1
Reference: URL:http://www.ubuntu.com/usn/USN-1983-1
Reference: UBUNTU:USN-1984-1
Reference: URL:http://www.ubuntu.com/usn/USN-1984-1
Reference: UBUNTU:USN-1985-1
Reference: URL:http://www.ubuntu.com/usn/USN-1985-1
Reference: SECUNIA:55107
Reference: URL:http://secunia.com/advisories/55107
Reference: SECUNIA:55116
Reference: URL:http://secunia.com/advisories/55116

Algorithmic complexity vulnerability in the ssl.match_hostname
function in Python 3.2.x, 3.3.x, and earlier, and unspecified versions
of python-backports-ssl_match_hostname as used for older Python
versions, allows remote attackers to cause a denial of service (CPU
consumption) via multiple wildcard characters in the common name in a
certificate.


Current Votes:
None (candidate not yet proposed)


